bedrock/firefox/templates/firefox/whatsnew/whatsnew-fxa.html

```diff
@@ -0,0 +1,56 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/new/quantum" %}
```

Comment:
This isn't going to work. For example: you're using `Take Firefox with You`, which is behind a tag in this lang file. Like this, we would end up enabling the page and showing the string in English for some locales.

We should use a separate lang file, and import the strings (`firefox/whatsnew_59`?)

Reply:
Let’s use `firefox/whatsnew_59`. Not only there's the tag/activation problem, but also a string that comes from a completely different file.